packages/chat/package.json

```diff
@@ -36,8 +36,13 @@
   "dependencies": {
     "@babel/runtime": "^7.23.2",
     "@carbon-labs/utilities": "0.4.0",
+    "@carbon/charts": "^1.15.5",
```

Comment:
Not sure if this is needed since I don't see `@carbon/charts` being used anywhere?